service/lotusclient/lotusclient.go

```diff
@@ -92,6 +93,33 @@ func (c *Client) HealthCheck() error {
 	return err
 }
 
+// CurrentSealingSectors returns the total number of sectors considered to be in sealing.
+func (c *Client) CurrentSealingSectors() (int, error) {
```

Comment:
Maybe worth `log.Debugf` to see how heavy this operation is.
I'm not sure how efficient `SectorSummary` is for miners that might have a lot of sectors.

Not sure we could do something about it, but at least we can have a sense on how much time this takes.